docs/html/development/issue-triage.rst

```diff
@@ -0,0 +1,305 @@
+
+* confirm bugs
+* provide support
+* discuss and design around the uses of the tool
+
+Specifically, to address an issue:
+
+#. Read issue title
+#. Scan issue description
+#. Ask questions
+#. If time is available, try to reproduce
+#. Search for or remember related issues and link to them
+#. Identify an appropriate area of concern (if applicable)
+
+Keep in mind that all communication is happening with other people and
+should be done with respect per the
+`Code of Conduct <https://www.pypa.io/en/latest/code-of-conduct/>`__.
+
+The lifecycle of an issue (bug or support) generally looks like:
+
+#. waiting for triage (marked with label ``triage``)
+#. confirming issue - some discussion with the user, gathering
+   details, trying to reproduce the issue (may be marked with a specific
+   category, ``S: awaiting-respose``, ``S: discussion-needed``, or
+   ``S: need-repro``)
+#. confirmed - the issue is pretty consistently reproducible in a
+   straightforward way, or a mechanism that could be causing the issue has been
+   identified
+#. awaiting fix - the fix is identified and no real discussion on the issue
+   is needed, should be marked ``R: awaiting PR``
+#. closed - can be for several reasons
+
+   * fixed
+   * could not be reproduced, no more details could be obtained, and no
+     progress can be made
+   * actual issue was with another project or related to system
+     configuration that pip cannot improve its behavior in the context
+     of
+
+
+Requesting information
+----------------------
+
+Requesting more information to better understand the context and environment
+that led to the issue. Examples of specific information that may be useful
+depending on the situation:
+
+* pip version: ``pip -V``
+* Python version: ``python -VV``
+* Python path: ``python -c 'import sys; print(sys.executable)'``
+* ``python`` on ``PATH``: Unix: ``which python``; Windows: ``where python``
+* Python as resolved by the shell: ``type python``
+* Origin of pip (get-pip.py, OS-level package manager, ensurepip, manual
+  installation)
+* Using a virtual environment (with `--system-site-packages`?)
+* Using a conda environment
+* ``PATH`` environment variable
+* Network situation (e.g. airgapped environment, firewalls)
+* ``--verbose`` output of a failing command
+* (Unix) ``strace`` output from a failing command (be careful not to output
+  into the same directory as a package that's being installed, otherwise pip
+  will loop forever copying the log file...)
+* (Windows)
+  `procmon <https://docs.microsoft.com/en-us/sysinternals/downloads/procmon>`__
+  output during a failing command
+  (`example request <https://github.com/pypa/pip/issues/6814#issuecomment-516611389>`__)
+* Listing of files relevant to the issue (e.g. ``ls -l venv/lib/pythonX.Y/problem-package.dist-info/``)
+* whether the unexpected behavior ever worked as expected - if so then what
+  were the details of the setup (same information as above)
```

Comment:
Maybe mention `pip debug` ?

Reply:
Added [here](https://github.com/pypa/pip/pull/6868/files#diff-0ee19f321d04652294c191edc037e9a7R157). Note that pip debug contains some of the information from the rest of the list, but I would like to leave deciding what to do about that for another PR. Considerations:

1. Do we maintain a list of the output fields here and in the `pip debug` implementation itself? (so we have to remember to update both)
1. If we're receptive to situations where someone cannot access the latest pip version (e.g. for the system-managed Python), then we would still maintain commands here for printing, for example `sys.executable` using plain Python.
1. Some of the "redundant" information can point to differences in the expected pip or Python being executed, which is another mark in favor of keeping the other commands in case we suspect `PATH` or the shell is selecting a different executable.